dataland-frontend/src/components/generics/ViewMultipleDatasetsDisplayBase.vue

```diff
@@ -21,11 +21,6 @@
             <h2 class="mb-0" data-test="frameworkDataTableTitle">{{ humanizeString(dataType) }}</h2>
           </div>
           <div class="col-12">
-            <EuTaxonomyForNonFinancialsPanel
```

Comment:
Mustn't there be a subsitute MLDTFrameworkPanel?

Reply:
Tatsächlich nicht.
Das unterste `<MultiLayerDataTableFrameworkPanel>` ist in der Lage, automatisch und dynamisch je nach url (und somit frameworkIdentifier) die korrekte ViewConfig zu besorgen und zu rendern.
D.h. für alle devtools-supporteten Frameworks braucht man hier im vue-file nichts weiter zu tun!

Die ganzen expliziten `<MultiLayerDataTableFrameworkPanel>` -tags die du hier siehst sind nur da, weil diese Frameworks MLDT-support haben, aber keinen devtool-support.

(SmePanel hat weder MLDT- noch devtool support und hat daher sogar ein eigenes Panel)

Reply:
Ach lit. Da habe ich wohl nicht genau genug hingeschaut.